### PR TITLE
chore: inline disable only MD011 issue

### PR DESCRIPTION
**Pull request by nschonni**
There is only one instance that this rule is flagging. Since Markdownlint doesn't try and work around mixed HTML/MD syntax it is unlikely a case that would be resolved upstream, so disabling inline seems best

### CODE REVIEW

File: files/en-us/web/javascript/guide/meta_programming/index.md [hunk 1]
```diff
@@ -285,7 +286,7 @@ The following table summarizes the available traps available to `Proxy` objects.
           </dd>
           <dt>Inherited property assignment</dt>
           <dd>
-            <code>Object.create(<var>proxy</var>)[foo] = bar</code
+            <code>Object.create[<var>proxy</var>](foo) = bar</code
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/guide/meta_programming/index.md:289]: I don’t understand what this is doing, or why.

It’s taking something that now renders as this:

```js
Object.create(proxy)[foo] = bar
```
…and it’s making it instead render as this:
```js
Object.create[proxy](foo) = bar
```
So it’s changing the syntax.

Why?

On the face of it, that doesn’t seem right.

Am I missing something here?

Comment (nschonni) [reply files/en-us/web/javascript/guide/meta_programming/index.md:289]:
```suggestion
            <!-- markdownlint-disable-next-line MD011 -->
            <code>Object.create(<var>proxy</var>)[foo] = bar</code
```

Comment (nschonni) [reply files/en-us/web/javascript/guide/meta_programming/index.md]: I missed that the `--fix` had changed this line too, so undid and added the same ignore as above

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/javascript/guide/meta_programming/index.md (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/javascript/guide/meta_programming/index.md]: > I missed that the `--fix` had changed this line too, so undid and added the same ignore as above

aha — I see now

**Comment by nschonni** [comment]
@DavidAnson does this look like a bug with parsing the HTML, or is this something because the VSCode/CLI1 is behind in the library?

**Comment by nschonni** [comment]
Opened up https://github.com/mdn/content/pull/12319 for now with an explicit enable/disable scope for now

**Comment by DavidAnson** [comment]
Sorry about that! A quick look doesn't tell me why this didn't work like you expected. I'll leave this in my notifications as a reminder to have a look soon. Glad you got it sorted out!

**Comment by DavidAnson** [comment]
@nschonni I tried dumping the raw version of https://github.com/mdn/content/blob/7534e0a7f594fd797f5893543e4299554bde0984/files/en-us/web/javascript/guide/meta_programming/index.md into https://dlaa.me/markdownlint/ just now and see only MD033 violations. Also, this PR was merged. Does that mean the issue was resolved? I'd not, is there a different version of the file I should try or do you have a small reproduction?

**Comment by nschonni** [comment]
@DavidAnson I'm not seeing on the live site either. I'll try again in VSCode/Markdown-cli1 though

**Comment by DavidAnson** [comment]
CLI2 is now on the latest library. VS Code extension coming soon.

**Comment by nschonni** [comment]
Yup, looks like it is just because the VSCode/CLI1 doesn't recognize the `markdownlint-disable-next-line` syntax right now, since I see it flagged for both those still. Workaround of fully disabling the rule works though for now

**Comment by DavidAnson** [comment]
Thanks for checking!

**Pull request merged**